importer/tag/src/test/java/org/itsallcode/openfasttrace/importer/tag/TestTagImporter.java

```diff
@@ -268,6 +268,29 @@ void tagWithMultipleRequiredCoverageWithSpaces()
                 itemWithReadableName(COVERING_ARTIFACT_TYPE1, 1, ID1, List.of(NEEDED_COVERAGE1, NEEDED_COVERAGE2)));
     }
 
+    // [utest->dsn~import.full-coverage-tag-with-revision~1]
+    @Test
+    void tagWithRevision()
+    {
+        assertItems(
+                tag("impl~~42", "req~example_name~17"), //
+                SpecificationItem.builder().id("impl", "example_name-4044529862", 42)
+                        .addCoveredId("req", "example_name", 17)
+                        .location(FILENAME, 1).build());
+    }
+
+    // [utest->dsn~import.full-coverage-tag-with-revision~1]
+    @Test
+    void tagWithRevisionAndNeededCoverage()
+    {
+        assertItems(
+                tag("impl~~42", "req~example_name~17", "test"), //
```

Comment:
```suggestion
                tag("impl~~42", "req~example_name~17", "test"), //
```
Here you see what I mean. This test is hard to decipher. If I didn't know about the forwarding feature, I would have zero clue what is tested here. Again, please spell the test out. You might even consider to provide the input as `ValueSource`, so that you can test a couple permutations for improved robustness.